dev-python/ipcalc/metadata.xml

```diff
@@ -1,14 +1,21 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
+	<longdescription lang="en">
+		IP subnet calculator
+	</longdescription>
 	<maintainer type="person">
-		<email>ercpe@gentoo.org</email>
```

Comment:
Why are you removing him? Did he resign or sth?

Reply:
I removed it for this discussion on ml:

https://archives.gentoo.org/gentoo-dev/message/c64f45e9fd507322b75e0bc2af814496

for what i understand this package is without a maintainer, is correct or no ? I don't know if I following right route. Sorry if not.

Reply:
Ok then, my bad. It seems undertakers haven't removed him from packages yet.